MetaMask/metamask-mobile
# test: approve erc721 token e2e

## **Description**

This PR is within the scope of the Quality Quest.

We're including an e2e test case to approve an ERC721 token.

**Test Steps:**
Given I am on the test dApp
When I tap on the Approve From button under the ERC 721 section
Then the transaction bottom sheet should appear
When I submit the transaction
Then the transaction should appear in the transaction history

## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to this page...
2.
3.

## **Screenshots/Recordings**


### **Before**


### **After**


## **Pre-merge author checklist**

- [✓] I’ve followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Mobile Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [✓] I've completed the PR template to the best of my ability
- [✓] I’ve included tests if applicable
- [✓] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [✓] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**
